src/games/strategy/engine/framework/ui/NewGameChooserModel.java

```diff
@@ -149,45 +152,33 @@ private static ZipProcessingResult processZipEntry(final URLClassLoader loader,
    * Open up a confirmation dialog, if user says yes, delete the map specified by
    * parameter, then show confirmation of deletion.
    */
-  private static void confirmWithUserAndThenDeleteCorruptZipFile(final File map, Optional<String> details) {
```

Comment:
The change is I think as intended. Not sure why the method signature line shows an update, that does not look changed. Otherwise we have a method that was previously wrapped entirely in a try/catch unnecessarily. Removing that, then worked in the library functions to remove the rest of the try/catches.